packages/shared/src/components/modals/common.tsx

```diff
@@ -7,7 +7,10 @@ export type CloseModalFunc = (
 ) => void;
 
 const SquadsBetaModal = dynamic(
-  () => import(/* webpackChunkName: "postToSquadModal" */ './SquadsBetaModal'),
```

Comment:
@rebelchris Was this intentional or a mistake?

Reply:
Ah that's a mistake! 👀